### DISCUSSION TITLE
Is Battle#16 4weeks or 3weeks long?

### DISCUSSION THREAD

**Discussion by Beowolve** [cssbattle/forum#157] [General]
The remaining time shows 14 days and we have 2 / 8 targets available at the moment. I think the battle startet with 21 days remaining, but the description says 4 weeks. So if we get only 2 targets every 7 days it will not work out very well for the last 2 targets 😃.
Last battle had 28 days total, am I missing something?

**Comment by rafehimself** [comment]
just 3 targets per week, with 1 open 😃

**Comment by chinchang** [comment]
Ahh, seems to be a bug...it's same 4 weeks only. We'll fix it. Thanks for
reporting :)

**Reply by chinchang** [reply]
this is fixed now :)
